field_rrule/demo/res_partner.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm fervently against that, but maybe I've a better offer: https://github.com/OCA/server-tools/pull/480/files#diff-9cc3eb31c10fb83195573350fb4de22bR8